src/test/resources/org/kuali/kfs/gl/spring-gl.xml

```diff
@@ -1157,7 +1157,7 @@
 	</bean>
 
 	<bean id="glPostExpenditureTransaction" parent="glPostExpenditureTransaction-parentBean" />
-	<bean id="glPostExpenditureTransaction-parentBean" class="org.kuali.kfs.gl.batch.service.impl.PostExpenditureTransaction" abstract="true">
+	<bean id="glPostExpenditureTransaction-parentBean" class="edu.cornell.kfs.gl.batch.service.impl.CuPostExpenditureTransaction" abstract="true">
```

Comment:
Do the unit tests load the "cu-spring-gl.xml" file? If so, then you may not need this change.